stdlib/public/runtime/MetadataCache.h

```diff
@@ -106,17 +106,14 @@ struct ConcurrencyControl {
   ConcurrencyControl() = default;
 };
 
-template <class EntryType, uint16_t Tag, bool ProvideDestructor = true>
+template <class EntryType, uint16_t Tag>
```

Comment:
Is there just never a situation where we care about destructors for these?

Reply:
I couldn't find any. `ConcurrentMap` doesn't support removal. It does support destroying the entire tree, but all uses of it just create it and let it live forever. I assume this was left over from a much earlier era when it was thought that these caches might not be permanent, or metadata might not be immortal.

Reply:
Mostly I was reluctant to write a class that might be useless in other contexts.

Reply:
Ah, that'll do it too.